### ISSUE TITLE
Include version in name of zip file

### ISSUE THREAD

**Issue by forumthemer** [forumthemer/eles-phpbb-forum-theme#35]
Currently, every new version of eles has the same name: `eles.zip`

It will be helpful for the users to include the version in the name of the zip file, something like eles-3.2.2.zip`. This will clear any confusion with updates.

**Comment by Leinad4Mind** [comment]
Agreed. For noobs it will be best. We can ofc check it on the configuration file... But it would be easier to have on the filename
